src/components/ComponentPages/Registration/UI/countryCodes.ts

```diff
@@ -0,0 +1,1212 @@
+export const country = [
```

Comment:
I think we should get this from the database it self.

Reply:
country list API implemented